pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -152,6 +159,8 @@ func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{Requeue: true}, nil
 	}
 
+	storage.Status.Conditions.Log(log)
```

Comment:
Why not just something like:
Log.Info("Conditions", "cnd", storage.Status.Conditions)

Reply:
that would provide one single log entry with all the conditions, I actually like to have a separate entry per condition to make it more readable. 

https://github.com/konveyor/mig-controller/pull/698#discussion_r511091777

Reply:
I think a single log entry is desirable because it is simple and more importantly keeps all of the conditions together.  With 6-7 controller logging, it will be easy for the list of conditions to be interlaced with other log entries and hard to read.  Also, if grammatically inspecting/digesting the logs, multiple entries will be more complex to correlate.

Reply:
```
With 6-7 controller logging, it will be easy for the list of conditions to be interlaced with other log entries and hard to read. 
```
With an increase in verbosity, devs will have to find a way to grep for CR's that they are looking for and interlacing logs among the CR is hard to see. if I am looking for storage controller logs I would almost always grep with `storage\|<cr-name>` to construct a timeline, I think with that in mind, a separate per line comment is more readable. I would do it like that, if you feel strongly, I can remove the method and log the entire blob of conditions inline